apps/vth-frontend/src/app/[locale]/page.tsx

```diff
@@ -23,19 +27,18 @@ export async function generateMetadata({ params: { locale } }: Params): Promise<
 }
 
 const Home = async ({ params: { locale } }: { params: any }) => {
-  const { t } = await useTranslation(locale, ['home-page', 'common']);
+  const { data } = await fetchData({
+    url: createStrapiURL(),
+    query: GET_HOME_PAGE,
+    variables: { locale: locale },
+  });
 
-  // todo: fetch theme data to display homepage cards
+  const { Title, Content } = data.homepage.data.attributes;
 
   return (
     <>
-      <Heading1>{t('h1')}</Heading1>
-      <p>
-        Wat pakt Toezicht en Handhaving in 2022 allemaal op? En welke resultaten hebben we de afgelopen periode bereikt?
-        Op deze website zie je in één oogopslag waarop we ons in 2023 richten, wat we de afgelopen jaren hebben gedaan
-        en welke trends we zien. Van de handhaving op afval tot het controleren van bouwplannen en van evenementen tot
-        de aanpak van huisjesmelkers.
-      </p>
+      <Heading1>{Title}</Heading1>
+      <Markdown>{Content}</Markdown>
```

Comment:
I you want to use images here, you have to provide this property `<Markdown strapiBackendURL={process.env.STRAPI_PUBLIC_URL}> </Markdown>` 

**Strapi:**

<img width="1455" alt="Screenshot 2023-08-31 at 16 28 03" src="https://github.com/frameless/strapi/assets/53454241/d3077c5b-dafb-49c4-9eaa-b1ca9217e363">

**Frontend**:
<img width="1070" alt="Screenshot 2023-08-31 at 16 28 24" src="https://github.com/frameless/strapi/assets/53454241/02b90691-9934-4d0c-a007-dad4399104b6">